pages/common/cbt.md

```diff
@@ -0,0 +1,28 @@
+# cbt
+
+> Utility for reading data from Google Cloud's Bigtable.
+> More information: <https://cloud.google.com/bigtable/docs/cbt-reference>.
+
+- List tables in the current project:
+
+`cbt ls`
+
+- Return count of rows in a specific table in the current project:
```

Comment:
`display` or `print`, `get` or `return` fit better in programming languages
```suggestion
- Print count of rows in a specific table in the current project:
```

Reply:
For shells it's better to not to use `return` term as there is no similar return functionality like in general purpose languages: POSIX-like shells return just exit codes, other info should be $(command-substituted).